backend/README.md

```diff
@@ -12,39 +12,54 @@ The backend is being developed with:
 * Spring Boot 2.2.5
 * Maven 3.6.0
 
-## Setup the local development environment
-
 We are using mostly IntelliJ IDE to develop the project. Currently, using
 **IntelliJ 2019.3.** 
 
-First make sure you have the correct versions of Java and Maven installed on your machine. Notice that IntelliJ bundles a version of Maven that might be different from your local version. This is not necessarily a project.
+## Setup the local development environment
+
+First make sure you have the correct versions of Java and Maven installed on your machine. Notice that IntelliJ bundles a version of Maven that might be different from your local version.
 
-Open up IntelliJ and select `Import Project`, then choose `Maven` and then
-choose the `backend` folder in this repository. The project should be imported correctly but the Maven window will probably display this failure:
+Clone the repo:
+```sh
+git clone https://github.com/ayudadigital/huelladigital-backend.git
+```
+
+Inside **IntelliJ** `New Project > Import Project From External Sources` (as Maven) and choose the `backend` folder in this repo.
+
+Maven might complain with:
 
     Failure to find junit:junit:pom:5.5.2 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced
 
-Don't worry the project should build, and the tests run. It's complaining because of the dependency tree but it's working.
+The project should build anyway, and the tests run. It's complaining because of the dependency tree but it works.
 
-### Plugins
+Make sure to set the Java compiler to v11: `Settings > Build, Execution, Deployment > Compiler > Java Compiler > Target bytecode version`
 
-In order to install plugins, go to `Settings > Build, Execution, Deployment > Compiler > Annotation Processors` and select `Enable annotation processing`.
```

Comment:
Don't miss the instructions to enable the annotation processors, they could be helpful for new collaborators.